### ISSUE TITLE
Performance on Open Graph Benchmark

### ISSUE THREAD

**Issue by EtoDemerzel0427** [mengliu1998/DeeperGNN#6]
Hi Meng,

[Open Graph Benchmark(OGB)](https://ogb.stanford.edu/) is gaining its popularity as benchmarks for GNNs, compared to citation networks we normally use, they are more scalable and systematic. I'd like to see DAGNN's results on OGB, and I did some experiments myself on [ogbn-arxiv](https://ogb.stanford.edu/docs/nodeprop/#ogbn-arxiv) by slightly modifying your code (making it work on SparseTensor). But I failed to reproduce a better accuracy compared to vanilla GCN. (GCN: 71.74%, while DAGNN only ~67%).

The Colab link is [here](https://colab.research.google.com/drive/1XvDjGgKl605boMntyQbKAlgfdK4N4w-3?usp=sharing). I tested K=5, 10, 20. Note that for consistency, I don't apply weight decay here since the GCN implementation doesn't either, and I don't think that would count for the 4% difference. Would you mind helping figure out what's wrong with my code or why this would happen? Thank you very much.

**Comment by mengliu1998** [comment]
@EtoDemerzel0427 

Thank you for your interest in our DAGNN.

I have submitted a request to access your Colab.

Actually, I also want and plan to evaluate DAGNN on OGB node-level datasets. BTW, I think it is reasonable to tune the hyperparameters for different datasets collected in different ways.

Thank you.

**Comment by EtoDemerzel0427** [comment]
Thanks for your immediate reply! I have approved the request and updated the sharing link above. Looking forward to your feedback.

As for tuning the hyperparameters, what kinds of hyperparameters do you mean? Besides the layers/hidden size of the MLP and K, I don't see much space to tune.

> @EtoDemerzel0427
> 
> Thank you for your interest in our DAGNN.
> 
> I have submitted a request to access your Colab.
> 
> Actually, I also want and plan to evaluate DAGNN on OGB node-level datasets. BTW, I think it is reasonable to tune the hyperparameters for different datasets collected in different ways.
> 
> Thank you.

**Comment by EtoDemerzel0427** [comment]
In case you need this, the Logger class in my code comes from this py file: [logger.py](https://github.com/snap-stanford/ogb/blob/master/examples/nodeproppred/arxiv/logger.py).

**Comment by mengliu1998** [comment]
Thank you.

Parameters about architecture (such as hidden and layer) and training (such as lr, wd, dropout, etc) should be tuned accordingly.

**Comment by EtoDemerzel0427** [comment]
> Thank you.
> 
> Parameters about architecture (such as hidden and layer) and training (such as lr, wd, dropout, etc) should be tuned accordingly.

Actually I followed the settings for lr, dropout, number of epochs of the baseline model(i.e GCN) provided by the OGB team. On this dataset, I also trained GAT, GraphSAGE, APPNP, JKNet, with these parameters, the results look reasonable. Anyways, it's still worth a try.

**Comment by mengliu1998** [comment]
The performance of our method DAGNN on ogbn-arxiv is uploaded and reproducible now.

More specifically, DAGNN achieves a competitive accuracy of 72.09% with only 43857 trainable parameters. It is parameter-efficient compared with other methods on this [leaderboard](https://ogb.stanford.edu/docs/leader_nodeprop/#ogbn-arxiv).

**Comment by EtoDemerzel0427** [comment]
Since this issue is well addressed, I will close this issue now. Thank you!
